packages/_site/src/pages/examples/index.tsx

```diff
@@ -20,8 +21,8 @@ const styles = stylex.create({
     marginTop: "1rem",
     border: "0.0625rem solid var(--border-color)",
     borderRadius: "0.25rem",
-    overflow: "hidden",
-    minHeight: "800px",
+    overflow: "auto",
+    minHeight: "1200px",
```

Comment:
Why was the height increased?

Reply:
My card was getting cut off for some reason, I tried messing around with the autoflow and the container but could not get it to show